src/seq_mv/csr_spgemm_device_rowest.c

```diff
@@ -19,9 +19,13 @@
  *- - - - - - - - - - - - - - - - - - - - - - - - - - */
 template <char type>
 static __device__ __forceinline__
-void rownnz_naive_rowi(HYPRE_Int rowi, HYPRE_Int lane_id, HYPRE_Int *ia, HYPRE_Int *ja,
+void rownnz_naive_rowi(HYPRE_Int  rowi,
```

Comment:
Should we add `hypre_` or `hypre_spgemm_` to the functions defined in this file?